### PR TITLE
LWIMIQ: Because you know I'm all about the bugs

### PR DESCRIPTION
**Pull request by juliancheal**

### CODE REVIEW

File: site/_posts/2017-11-21-im-all-about-the-bugs.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+author: juliancheal
+date: 2017-11-21
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+It's that time of year again what with; Thanksgiving, Black Friday deals, and many other seasonal holidays around the corner. On everyone's mind though before we get to all that are… …bugs. Lots and lots of bugs. Well more specifically bug fixes. With the Gaprindashvili release just around the corner. The team is busy busy, fixing bugs and making the bestest release of ManageIQ yet.
+
+This issue of Last Week in ManageIQ is brought to you this week by [Julian](https://juliancheal.co.uk). And along for the ride this week are [37 pull requests merged][manageiq PRs merged] in ManageIQ core [45 pull requests merged][manageiq-ui-classic PRs merged] in the UI-Classic, and many more in our plethora of repositories.
```
Comment (d-m-u) [inline site/_posts/2017-11-21-im-all-about-the-bugs.md]: :P `The owner of juliancheal.co.uk has configured their website improperly`

Comment (juliancheal) [reply site/_posts/2017-11-21-im-all-about-the-bugs.md]: Welp. I'll just un-`https` the link for now.

**Comment by d-m-u** [comment]
👍 lgtm, just minor nit: should "With the Gaprindashvili release just around the corner. The team..." the full stop be a comma, maybe?

File: site/_posts/2017-11-21-im-all-about-the-bugs.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Fixed
+
+'Tis the season for some bug fixes, fa-la-la-la-la, la-la-la-la.
+With this being bug fixing season, there are so many wonderful fixed bugs to choose from. For example naming is hard as we all know, so now by sorting the order of Amazon's queue names, this [fix](https://github.com/ManageIQ/manageiq-providers-amazon/pull/361) should help improve the Amazon workers this holiday season.
+
+Azure too got a [fix](https://github.com/ManageIQ/manageiq-providers-azure/pull/165) this week ending the tyranny of bugs including not being able to start or stop an Azure VM.
+
+## New
+
+Not so many new things this week what with it being bug fixing season. Did I mention that `Gaprindashvili-1-Beta1` is [out](http://manageiq.org/download/)
```
Comment (cybette) [inline site/_posts/2017-11-21-im-all-about-the-bugs.md]: Suggestion: link to the [blog post](http://manageiq.org/blog/2017/11/Announcing-Gaprindashvili-Beta1/) as there's already a link to download above. Also download page updates with each new release so the G-Beta1 won't always be there :)

Comment (juliancheal) [reply site/_posts/2017-11-21-im-all-about-the-bugs.md]: Ah good point!

**Comment by juliancheal** [comment]
Changed both links to point to the blog post.

**Pull request merged**